src/toil/job.py

```diff
@@ -2483,6 +2757,7 @@ def resolve(key, default=None, dehumanize=False):
         super().__init__(memory=resolve('memory', dehumanize=True),
                          cores=resolve('cores', dehumanize=True),
                          disk=resolve('disk', dehumanize=True),
+                         accelerators=resolve('accelerators'),
```

Comment:
I don't think we should do that; we want to be able to have jobs whose `accelerators` requirement is unset, so that `--defaultAccelerators` can do anything.

Though I just noticed I forgot to actually *add* `--defaultAccelerators`.